content/intro-to-storybook/angular/es/get-started.md

```diff
@@ -1,56 +1,79 @@
 ---
-title: 'Storybook para o Angular tutorial'
+title: 'Tutorial de Storybook para Angular'
 tocTitle: 'Empezando'
 description: 'Configurar Angular Storybook en tu entorno de desarrollo'
 commit: 0818d47
 ---
 
-Storybook se ejecuta en conjunto con tu aplicación en modo desarrollo. Te ayuda a crear componentes de interfaz gráfica aislados de la lógica y el contexto de tu aplicación. Esta edición de Aprende Storybook es para Angular; existe otras ediciones para [React](/intro-to-storybook/react/es/get-started), [React Native](/intro-to-storybook/react-native/es/get-started), [Vue](/intro-to-storybook/vue/es/get-started) y [Svelte](/intro-to-storybook/svelte/es/get-started).
+Storybook se ejecuta junto con su aplicación en modo de desarrollo. Le ayuda a crear componentes de interfaz de 
+usuario aislados de la lógica de negocio y el contexto de su aplicación. Esta edición del tutorial de Storybook es para 
+Angular;
+existen otras ediciones para [React](/intro-to-storybook/react/en/get-started), [React Native](/intro-to-storybook/react-native/en/get-started), [Vue](/intro-to-storybook/vue/en/get-started), [Svelte](/intro-to-storybook/svelte/en/get-started) y [Ember](/intro-to-storybook/ember/en/get-started).
 
 ![Storybook and your app](/intro-to-storybook/storybook-relationship.jpg)
 
-## Configurando Angular Storybook
+## Configurar Angular Storybook
 
-Necesitaremos seguir algunos pasos para configurar el proceso de transformación y agregación de recursos y archivos en nuestro entorno. Para esto, vamos a utilizar [@angular/cli](https://cli.angular.io/). Adicionalmente, añadiremos [Storybook](https://storybook.js.org/) y [Jest](https://facebook.github.io/jest/) para probar nuestra aplicación. Para hacerlo, vamos a ejecutar los siguientes comandos:
+Necesitaremos seguir algunos pasos para configurar el proceso de transformación y agregación de recursos y archivos en nuestro entorno. Para empezar, vamos 
+a usar [degit](https://github.com/Rich-Harris/degit) para configurar nuestro sistema de construcción. Usando este 
+paquete, podrás descargar
+"templates" (aplicaciones parcialmente construidas con alguna configuración predeterminada) para ayudarte a acelerar 
+tu flujo de trabajo de desarrollo.
+
+Ejecutemos los siguientes comandos:
 
 ```bash
-# Crea nuestra aplicación:
-npx ng new taskbox --style less
+# Clonar la plantilla
+npx degit chromaui/intro-storybook-angular-template taskbox
+
 cd taskbox
 
-# Añade Storybook:
-npx -p @storybook/cli sb init
+# Instalar dependencias
+npm install
 ```
```

Comment:
@josegarciamanez If you could leave the code comments in their native form (English) throughout the tutorial,we'd appreciate it. I'm aware that translating them adds context and brings the translation closer to home but until we have a good solution for this we recommend that they remain as mentioned.